content/general/concepts/hexadecimal/hexadecimal.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: 'Hexadecimal'
+Description: 
```

Comment:
We're missing a description in the metadata. (You can paraphrase the first sentence of your entry.)